packages/testing/src/TestUtils.ts

```diff
@@ -330,15 +327,27 @@ export async function createFullDidFromLightDid(
   lightDidForId: LightDidDetails,
   sign: SignCallback
 ): Promise<FullDidDetails> {
-  const api = await BlockchainApiConnection.getConnectionOrConnect()
-  return FullDidCreationBuilder.fromLightDidDetails(api, lightDidForId)
-    .setAttestationKey(lightDidForId.authenticationKey)
-    .setDelegationKey(lightDidForId.authenticationKey)
-    .buildAndSubmit(sign, payer.address, async (tx) => {
-      await Blockchain.signAndSubmitTx(tx, payer, {
-        resolveOn: Blockchain.IS_IN_BLOCK,
-      })
-    })
+  const { authenticationKey } = lightDidForId
+  const tx = await Chain.generateCreateTxFromCreationDetails(
```

Comment:
as mentioned on the other PR: kind of a long and convoluted name for a function you will now have to use to make a full did.

Reply:
I agree and I want to change it!

Reply:
if that stays to be the main function people will have to use to upgrade to an on-chain did, would it also make sense to put it rather in the DID, or FullDID namespace?

Reply:
I guess it should follow the pattern of `getStoreTx` functions for other data objects. For the CType and Attestation the methods from the chain file are exported on the top level. For whatever reason the methods from the DID chain file are only exported in the Chain sub-namespace. Should I re-export just `generateCreateTxFromCreationDetails` and `generateCreateTxFromDidDetails` in the DID namespace? Or should it be done in some next PR?

Reply:
I’ve renamed it in 28659889 in the other PR